dev/com.ibm.ws.webcontainer/src/com/ibm/ws/webcontainer/osgi/WebContainer.java

```diff
@@ -236,6 +236,14 @@ protected void unsetWebMBeanRuntime(ServiceReference<WebMBeanRuntime> ref) {
     private URIMatcherFactory uriMatcherFactory;
     
     
+    public static final int SPEC_LEVEL_30 = 30;
+    public static final int SPEC_LEVEL_31 = 31;
+    public static final int SPEC_LEVEL_40 = 40;
+    public static final int SPEC_LEVEL_50 = 50;
```

Comment:
I'm not sure why these constants are defined here when these values come from the properties file loaded.  I see they may be used by tests.  Should the be defined in the tests then?

This is a picky comment that you can choose to ignore.

Reply:
We use these variables elsewhere in the code. 
For example, https://github.com/OpenLiberty/open-liberty/blob/2fd4a880754c37a988c5ed9ac4f1ea5988e465d6/dev/com.ibm.ws.webcontainer.servlet.4.0/src/com/ibm/ws/webcontainer40/srt/SRTServletResponse40.java#L80

I'll keep these in.